viperx/vars_test.go

```diff
@@ -23,3 +23,41 @@ func TestUnmarshalKey(t *testing.T) {
 	assert.EqualValues(t, true, config.Enabled)
 	assert.EqualValues(t, `{"providers":[{"client_id":"kratos-client","client_secret":"kratos-secret","id":"hydra","issuer_url":"http://127.0.0.1:4444/","mapper_url":"file://test/e2e/profiles/oidc/hydra.jsonnet","provider":"generic","scope":["offline"]},{"client_id":"google-client","client_secret":"kratos-secret","id":"google","issuer_url":"http://127.0.0.1:4444/","mapper_url":"file://test/e2e/profiles/oidc/hydra.jsonnet","provider":"generic","scope":["offline"]},{"client_id":"github-client","client_secret":"kratos-secret","id":"github","issuer_url":"http://127.0.0.1:4444/","mapper_url":"file://test/e2e/profiles/oidc/hydra.jsonnet","provider":"generic","scope":["offline"]}]}`, string(config.Config))
 }
+
+func TestUnmarshalKeyWithJsonObject(t *testing.T) {
```

Comment:
We usually add these test cases using

```go
func TestUnmarshalKey(t *testing.T) {
    t.Run("case=with JSON object", func(t) {
        ...
    })
}
```

Would be awesome if you could do that as well :wink: